plugins/bazel/cargo/Cargo.toml

```diff
@@ -10,8 +10,9 @@ proxy-wasm = "0.2.1"
 log = "0.4"
 regex = "1.9"
 url = "2.4"
+# Pin version 2.0.0 until rust-version includes https://github.com/rust-lang/rust/issues/119128
```

Comment:
That PR is a year old and should be in Rust stable (1.85.0, branched on 3 January, 2025). Have you tried updating Rust in this repo to see if the latest lol-html works? If it's a larger project please file an Issue to track the update (and mention the getrandom issue at this version, since an update might fix both).

Reply:
We inherit the Rust version specified in [Proxy-wasm-cpp-host](https://github.com/proxy-wasm/proxy-wasm-cpp-host/blob/main/bazel/dependencies.bzl#L42) which is "1.77.2". I filed an issue to track this. https://github.com/proxy-wasm/proxy-wasm-cpp-host/issues/429